common/workunit/workflow.cpp

```diff
@@ -80,18 +82,18 @@ The actions in an ordered actionlist have a less constrained ordering requiremen
 In relation to the workflow engine, the second action in the actionlist has a logical dependency on the first action. This is not true of the second action's dependencies, which can be executed at any point.
 
 What makes an item active?
-Any actions explicitly called in the ECL query are active. This encodes the idea that for the query to succeed, it is necessary for this item to execute.
-Any items that these actions depend on automatically get marked as active.
+Any actions explicitly called in the ECL query are active. For the query to succeed, it is always necessary for these item to execute.
+Any items that these actions depend on also get marked as active.
 Items that start without being active (i.e. those that might not execute during the workunit) are:
-•	those that are contingencies (SUCCESS/FAILURE)
-•	those that are the trueresult or falseresult of an IF function
+•	items that are contingencies (SUCCESS/FAILURE)
+•	items that are the trueresult or falseresult of a conditional IF function
 If one of these items is going to execute, then they are marked as active.
```

Comment:
* items that are logical successors of other items, and not activated by another route.